### PR TITLE
User interface started

### PR DESCRIPTION
**Pull request by Igors78**
In this project, the student builds a Tic Tac Toe game. The main goal is to put into practice the main concepts of Object-Oriented Programming. Particularly, classes and access to their attributes. Created two directories lib and bin, placed the main executable file in the bin folder. Do not need to write an extension. Displayed prompts for users, collect user input and only mock the basic flow of the game.

### CODE REVIEW

File: bin/main [hunk 1]
```diff
@@ -1,7 +1,91 @@
...
-  puts "Hello, darling!"
+puts "Tic-tac-toe (American English), noughts and crosses (Commonwealth English),
+ or Xs and Os, is a paper-and-pencil game for two players, X and O, who take
+ turns marking the spaces in a 3×3 grid. The player who succeeds in placing
+ three of their marks in a horizontal, vertical, or diagonal row is the winner.
+ It is a solved game with a forced draw assuming best play from both players.
+ Please follow the prompt to enter your move.
+ Your cross (X) or your (O) will be placed in the cell with corresponding number you enter,
+ check reference board.
+ Press ENTER to start"
```
Comment (mahmoud717) [inline bin/main:10]: you don't have to include this information in the start of the game, you will later put it in the readme file.

File: bin/main [hunk 2]
```diff
@@ -1,7 +1,91 @@
...
+      player2 = gets.chomp.to_i
+      check_entry(player2)
+    end
+    system "clear"
+    start_game
+  end
+
+  def check_entry(entry)
+    if entry > 9 || entry < 1 || !entry.is_a?(Numeric)
+      puts "Range from 1 to 9"
```
Comment (mahmoud717) [inline bin/main:54]: please use a more descriptive error message like " invalid move, please choose a number between 1 , 9 "
to let the user know that they have made a mistake

**Review by mahmoud717 — CHANGES_REQUESTED** [review]
### Changes required
**hi @Igors78 **
you have done a really great job, still there are some changes to be made
### Required 
please check the inline comments

**Review by ebeagusamuel — APPROVED** [review]
# Project Approved :100: 

You have done well in implementing the changes requested by the previous TSE. Well done in completing this milestone :+1: .
You can go ahead and merge this branch.
Good luck with your next project.

slack(@ebeagusamuel)

**Pull request merged**